### PR TITLE
Overhaul

### PR DESCRIPTION
**Pull request by ryan-odea**
Overhaul of package - including passing the CMD check
Potential breaking changes were moving the package folder one level down, otherwise just refreshed documentation

### CODE REVIEW

File: Syndemics/R/fetch_sas.R [hunk 1]
```diff
@@ -1,13 +1,16 @@
 #' MA DPH PHDW Script Pull
 #' Fetches most recent .SAS files for MA DPH work and saves them to a SAS/ subdirectory
 #'
-#' @importFrom httr GET
+#' @import httr
 #'
 #' @export
 fetch_sas <- function(){
   if(!dir.exists("SAS")) dir.create("SAS")
+  req <- httr::GET("https://api.github.com/repos/SyndemicsLab/PHD/git/trees/main?recursive=1")
```
Comment (MJC598) [inline Syndemics/R/fetch_sas.R:9]: This only pulls from main. Just noting in the condition that you might want to pull sas from a different branch like PreVenT?

File: Syndemics/R/internalTools_crc.R [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  })
+
+  non_interaction_vars <- setdiff(rownames(corr_matrix), c(interactions_df$var1, interactions_df$var2))
+  formula_string <- paste(paste(non_interaction_vars, collapse = " + "), "+", paste(interaction_terms, collapse = " + "))
+  formula_string <- paste(freq.column, "~", formula_string)
+
+  formula_object <- as.formula(formula_string)
+
+  return(formula_object)
+}
```
Comment (ddbaptiste) [inline Syndemics/R/internalTools_crc.R:122]: Is there a difference between these lines and lines 58-89?

**Review by ddbaptiste — DISMISSED** [review]
Looks good! I'm approving, but my review points out a couple places where it seems that the same content might've been duplicated.

File: Syndemics/R/internalTools_crc.R [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+#' Internal function to create a formula based on correlation among databases listed in \code{CRC}
+#'
+#' @param corr_matrix A correlation matrix
+#' @param threshold Threshold for interacting two databases
+#' @param freq.column frequency column, e.g. N_ID
+#'
+#' @keywords internal
+formula_corr <- function(corr_matrix, threshold, freq.column){
+  corr_upper <- corr_matrix; corr_upper[upper.tri(corr_matrix, diag = TRUE)] <- NA
```
Comment (MJC598) [inline Syndemics/R/internalTools_crc.R:66]: Any particular reason this is on one line? It is a bit tricky to read

File: Syndemics/R/internalTools_crc.R (hunk 2 above)
Comment (MJC598) [inline Syndemics/R/internalTools_crc.R:122]: This looks like a duplicate of lines 58-89?

File: Syndemics/R/internalTools_crc.R
Comment (MJC598) [inline Syndemics/R/internalTools_crc.R:1]: Can we maintain a file naming convention? We use both camel and snake case here

**Review by MJC598 — CHANGES_REQUESTED** [review]
A couple questions. Overarching though, we should probably maintain a file naming convention. Does R prefer camel or snake case?

File: Syndemics/R/fetch_sas.R (hunk 1 above)
Comment (ryan-odea) [reply Syndemics/R/fetch_sas.R:9]: I can add this condition, but as this is a RESPOND package + CRC, does PreVenT even need to be included?

File: Syndemics/R/internalTools_crc.R (hunk 2 above)
Comment (ryan-odea) [reply Syndemics/R/internalTools_crc.R]: Removed! Thanks for flagging

File: README.md [hunk 4]
```diff
@@ -1,2 +1,10 @@
 # Syndemics
+  <!-- badges: start -->
+  [![CRAN status](https://www.r-pkg.org/badges/version/Syndemics)](https://CRAN.R-project.org/package=Syndemics)
+  ![CRAN Downloads](https://cranlogs.r-pkg.org/badges/grand-total/Syndemics)
+  [![R-CMD-check](https://github.com/SyndemicsLab/Syndemics/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/SyndemicsLab/Syndemics/actions/workflows/R-CMD-check.yaml)
+  [![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
+  <!-- badges: end -->
+Collection of functions developed by the Boston Medical Center Syndemics Lab for use interacting with the RESPOND model.
+
 Collection of functions developed by the Boston Medical Center Syndemics Lab
```
[the remark this answers is not in the document]
Comment (ryan-odea) [reply README.md]: Removed!

File: Syndemics/R/internalTools_crc.R (hunk 3 above)
Comment (ryan-odea) [reply Syndemics/R/internalTools_crc.R]: I was probably thinking that the corr_matrix 'setup' would be easiest to denote on one line - this has been changed

Comment (MJC598) [reply Syndemics/R/internalTools_crc.R]: Resolved: [a2d2dae](https://github.com/SyndemicsLab/Syndemics/pull/8/commits/a2d2daefb720df55019d2c12d612298b82c16393)

File: Syndemics/R/internalTools_crc.R
Comment (MJC598) [reply Syndemics/R/internalTools_crc.R:1]: Convention is set to have {{project}}_{{descriptionName}}.R. Resolved.

**Review by MJC598 — APPROVED** [review]
Thanks for the changes!

**Pull request merged**